packages/storybook/src/nimble/table-column/menu-button/table-column-menu-button.stories.ts

```diff
@@ -0,0 +1,187 @@
+    decorators: [withActions<HtmlRenderer>],
+    parameters: {
+        actions: {
+            handles: [
+                ...sharedTableActions,
+                'menu-button-column-toggle',
+                'menu-button-column-beforetoggle'
+            ]
+        }
+    },
+    // prettier-ignore
+    argTypes: {
+        ...sharedTableArgTypes,
+        selectionMode: {
+            table: {
+                disable: true
+            }
+        },
+    },
+    args: {
+        ...sharedTableArgs(simpleData)
+    }
+};
+
+export default metadata;
+
+interface MenuButtonColumnTableArgs extends SharedTableArgs {
+    fieldName: string;
+    menuSlot: string;
+    updateMenuItems: (
+        storyArgs: MenuButtonColumnTableArgs,
+        e: CustomEvent<MenuButtonColumnToggleEventDetail>
+    ) => void;
+    menuRef: Menu;
+    headerContent: string;
+    toggleEvent: never;
+    beforeToggleEvent: never;
+}
+
+export const menuButtonColumn: StoryObj<MenuButtonColumnTableArgs> = {
+    parameters: {},
+    // prettier-ignore
+    render: createUserSelectedThemeStory(html<MenuButtonColumnTableArgs>`
+        <${tableTag}
+            ${ref('tableRef')}
+            data-unused="${x => x.updateData(x)}"
+            id-field-name="id"
+        >
+            <${tableColumnTextTag} field-name="firstName">
+                First Name
+            </${tableColumnTextTag}>
+            <${tableColumnTextTag} field-name="lastName">
+                Last Name
+            </${tableColumnTextTag}>
+            <${tableColumnMenuButtonTag}
+                field-name="${x => x.fieldName}"
+                menu-slot="${x => x.menuSlot}"
+                @menu-button-column-beforetoggle="${(x, c) => x.updateMenuItems(x, c.event as CustomEvent<MenuButtonColumnToggleEventDetail>)}"
+            >
+                ${x => x.headerContent}
+            </${tableColumnMenuButtonTag}>
+
+            <${menuTag} ${ref('menuRef')} slot="${x => x.menuSlot}">
+            </${menuTag}>
+        </${tableTag}>
+    `),
+    argTypes: {
+        fieldName: {
+            name: 'field-name',
+            description:
+                "Set this attribute to identify which field in the data record contains the visible text value for each cell's menu button in the column. The field values must be of type `string`.",
+            control: false,
+            table: { category: apiCategory.attributes }
+        },
+        menuSlot: {
+            name: 'menu-slot',
+            description:
+                "The name of the slot within the `nimble-table` instance where the menu associated with the column's menu button will be provided",
+            control: false,
+            table: { category: apiCategory.attributes }
+        },
+        updateMenuItems: {
+            table: {
+                disable: true
+            }
+        },
+        menuRef: {
+            table: {
+                disable: true
+            }
+        },
+        headerContent: {
```

Comment:
We don't typically document the default slot for header content on other column types since it's covered by the Table Column Configuration story. Is there any reason to diverge from that pattern for this column?

Reply:
I didn't realize that it was an established pattern. It just seemed to me that it had been left off by accident on all the columns. I don't mind removing it from here if we think the behavior is documented sufficiently in a different story that applies to all columns.

Reply:
I took a stab at addressing this feedback in #2231. If we like that approach and it goes in before this PR, you'll need to update this column's MDX file to refer to the same shared link text.

Reply:
Updated to follow the pattern established in your PR.